### ISSUE TITLE
Provide default java implementation for TMK

### ISSUE THREAD

**Issue by lfcnassif** [facebook/ThreatExchange#210]
Hello,

Like there is a default java implementation for PDQ, I think it makes sense having one for TMK. Maybe a JNI/JNA wrapper may be sufficient.

**Comment by johnkerl** [comment]
Thanks! Another partner has asked for this as well ... at presented we're looking to schedule time to do the port. Java is always a popular language for folks to use! :)

**Comment by lfcnassif** [comment]
I still think it could be useful. Thanks for the project. Unfortunatelly I currently don't have time to contribute...

**Comment by bodnarbm** [comment]
Thanks for updating, @lfcnassif! I am not sure when our team will be able to work on porting the code to java, but I will mark this issue with a help-wanted tag in case anyone with free cycles is interested in doing the port (likely through a JNI or JNA wrapper as you suggested). Really appreciate the suggestion and update.

**Comment by BigPanda97** [comment]
I agree here, a Java implementation of TMK would be really nice. It is requested for nearly 2 years now…
